chess/src/game.rs

```diff
@@ -36,6 +36,7 @@ pub enum GameStatus {
     Check,
     Checkmate,
     Replay,
+    Draw,
```

Comment:
So a `Stalemate` is also a draw, so maybe we either have a single status for all draws, or name this something specific to the 50 move rule.

Reply:
ok perfectoo